flutter/realm_flutter/example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:7.3.0'
```

Comment:
After increasing the version, there is a compile message in Android studio
```
The option 'android.enableR8' is deprecated.
It was removed in version 7.0 of the Android Gradle plugin.
Please remove it from `gradle.properties`.
```

Reply:
that's a warning coming from our flutter\example. I don't think it is crucial to anything